tests/docker/Dockerfile

```diff
@@ -21,7 +21,7 @@ ENV MIRROR="http://apache.cs.utah.edu/"
 
 ADD ssh /root/.ssh
 RUN chmod 600 /root/.ssh/id_rsa
-RUN apt update && apt install -y unzip wget curl jq coreutils openssh-server net-tools vim openjdk-8-jdk python-pip python-dev libffi-dev libssl-dev
+RUN apt update && apt install -y unzip wget curl jq coreutils openssh-server net-tools vim python-pip python-dev libffi-dev libssl-dev
```

Comment:
Seems like removing the `openjdk-8-jdk` install will break things for anyone running on docker outside travis? Or I'm missing something?

Perhaps more helpfully, does it break if we still include the jdk so it will work for non-Travis cases?